insn-riscv/cgetaddr.tex

```diff
@@ -14,3 +14,7 @@ \subsubsection*{Format}
 \end{center}
 
 \sailRISCVisarefbody{CGetAddr}
+
+\subsubsection*{Deprecated}
```

Comment:
CGetAddr has also been removed from the compiler, it now emits addi 0 instead.

Reply:
Yes, it will expand it to an `ADDI 0` (`MV`).

Reply:
Ok, I've updated the series to remove CGetAddr entirely.